semopenalex-app/data/templates/Template%3Ahttps%3A%2F%2Fsemopenalex.org%2Fclass%2FInstitution.html

```diff
@@ -158,10 +158,10 @@
             <{{page-resource}}> <https://dbpedia.org/ontology/location> ?location .
             ?location <http://www.geonames.org/ontology#lat> ?lat .
             ?location <http://www.geonames.org/ontology#long> ?lng .
-            ?location <https://dbpedia.org/property/city> ?city .
-            ?location <https://dbpedia.org/property/country> ?country .
+            OPTIONAL { ?location <https://dbpedia.org/property/city> ?city . }
+            OPTIONAL { ?location <https://dbpedia.org/property/country> ?country .}
           }' 
-          tuple-template='<b>{{city.value}} ({{country.value}})</b>'>
+          tuple-template='<b>{{#if city}}{{city.value}}{{/if}} {{#if country}}({{country.value}}){{/if}}</b>'>
```

Comment:
minor detail: this will produce an empty `<b>` block if neither city nor country is bound